src/tests/resource.test.ts

```diff
@@ -52,35 +61,43 @@ describe('ResourceTests', () => {
 
     test('Add child to resource', async () => {
         const res = await api.resources.add_child({
-            'child-name': 'testChild',
-            'parent-name': 'testResc',
+            'child-name': 'testChildResc',
+            'parent-name': 'testParentResc',
```

Comment:
Consider replacing the hard-coded resource name strings with variables.

Reply:
Done.